app/views/users/_created.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="p-4">
+
+<% if @created_by_user.empty? %>
+  <%= "There're no events created by #{@user.name}" %>
+<% else %>
+<% @created_by_user.each do |event| %>
+<div class="card mb-2 " style="width: 18rem;">
+  <div class="card-body">
+  <h5 class="card-title"><%= "#{@user.name} created an event in #{event.location} " %> </h5>
+  <p class="card-text"> <%= "Event date: #{event.date}." %></p>
+  <%= link_to 'View event', event_show_path(event)  %>
+  </div>
+</div>
+<% end %>
+<% end %>
```

Comment:
- Move the if/else statement to a helper method that you call here and render the markup within the helper method using a partial